tests/network/test_ocrd_all_workflow.py

```diff
@@ -0,0 +1,22 @@
+from src.ocrd_network.models import StateEnum
+from tests.network.config import test_config
+from tests.network.utils import (
+    poll_job_till_timeout_fail_or_success,
+    post_ps_workflow_request,
+)
+
+PROCESSING_SERVER_URL = test_config.PROCESSING_SERVER_URL
+
+
+def test_ocrd_all_workflow():
+    # This test is supposed to run with ocrd_all not with just core on its own
+    # Note: the used workflow path is volume mapped
+    path_to_wf = "/ocrd-data/assets/ocrd_all-test-workflow.txt"
```

Comment:
Why the `ocrd-data` prefix, why not just use `/data`?

Reply:
I used `/ocrd-data` instead of `/data` as well, [here](https://github.com/OCR-D/core/blob/refactor-network/tests/network/docker-compose.yml). My reasoning at least was to avoid potential accidental overwriting of an already existing `/data` folder coming from the `ocrd_all` image.

Reply:
I use a mounted docker volume (not host directory) to transfer the test-workspace. I didn't want to interfere with /data because that is usally used to mount a host folder to the container. But maybe it is better to use data, I didn't try that yet.